### ISSUE TITLE
Cannot read properties of undefined (reading 'left') with Zoom Component and React SVG Component in Docusaurus

### ISSUE THREAD

**Issue by LichLord91** [rpearce/react-medium-image-zoom#406]
## Issue Type

- [x] Bug report
- [ ] Feature request
- [ ] Question / support request
- [ ] Other

## Description

react-medium-image-zoom version 5.1.3

I'm using Docusaurus 2.4.0 and when I'm zooming in/out of the page with a Zoom component and a SVG inside , Docusarus will crash eventually with 

![image](https://user-images.githubusercontent.com/8435580/232273811-fd46242f-571a-453f-addd-1384f50e52c0.png)

![image](https://user-images.githubusercontent.com/8435580/232273862-df38f173-058b-413c-bdff-2c147375bf59.png)

code is at the end of https://github.com/rpearce/react-medium-image-zoom/blob/main/source/utils.ts

![image](https://user-images.githubusercontent.com/8435580/232273904-eae7fedb-2f53-482b-b57c-42b4dbe181e5.png)

Place detailed information here
fsdf

**Comment by rpearce** [comment]
@LichLord91 Thanks for opening your issue and asking for assistance! Could you please provide an example image for me to test with? The ideal thing to do would be to provide a Codesandbox or example repository, but if all you can provide is an image, that would be helpful.

**Comment by rpearce** [comment]
For starters, I'm thinking altering that logic to be this may be the fix, but I'd like to get a testable scenario from you.

```js
export const getStyleGhost: GetStyleGhost = (imgEl) => {
  if (!imgEl) {
    return {}
  }

  if (testSvg(imgEl)) {
    const parentEl = imgEl.parentElement
    const rect = imgEl.getBoundingClientRect()

    if (parentEl) {
      const parentRect = parentEl.getBoundingClientRect()

      return {
        width: rect.width,
        height: rect.height,
        top: parentRect.top - rect.top,
        left: parentRect.left - rect.left,
      }
    } else {
      return {
        width: rect.width,
        height: rect.height,
        top: rect.top,
        left: rect.left,
      }
    }
  } else {
    return {
      width: imgEl.offsetWidth,
      height: imgEl.offsetHeight,
      top: imgEl.offsetTop,
      left: imgEl.offsetLeft,
    }
  }
}
```

**Comment by rpearce** [comment]
It seems this code was unwise to begin with, so I may merge it before I hear back from you. Here's the PR: https://github.com/rpearce/react-medium-image-zoom/pull/407

**Comment by rpearce** [comment]
Can you please 🙏 try this fix out? `npm i react-medium-image-zoom@5.1.4-rc.0`

**Comment by LichLord91** [comment]
@rpearce  that seemed to do the trick no crashing on the 5.1.4-rc.0!
